R/setOrRetrieve.R

```diff
@@ -0,0 +1,108 @@
+.internal <- list2env(list(
+  recomputedHashtab = hashtab(type = "address", NULL), # unclear if "address" is the correct choice here
+  lastRecomputed    = TRUE
+), parent = emptyenv())
+
+saveHashOfJaspObject <- function(x) {
+  .internal[["recomputedHashtab"]][[x]] <- 0L
+}
+
+setRecomputed <- function(x) {
+  .internal[["lastRecomputed"]] <- x
+}
+
+#' Test if a jaspObject was recomputed of not.
+#' @param x a jaspObject, or missing in which case the last created or retrieved jaspObject is considered.
+#'
+#' @details Note that this function only works when \code{\link{\%setOrRetrieve\%}} is used to set or retrieve a jaspObject.
```

Comment:
any reason to use `\code{\link{\%setOrRetrieve\%}}` instead of `[%setOrRetrieve]`? The latter is more readable to me.

Reply:
`[%setOrRetrieve]` does not create a link to the helpfile or does it? Either way, I merged the helpfiles so I replaced everything by \`%setOrRetrieve%\`.

Reply:
> [%setOrRetrieve] does not create a link to the helpfile or does it? 

it does, if it's referencing a function in the same package (otherwise you need to prefix with `package::`). The link won't show up properly if you render the help files using `devtools::document()`, but if you install normally it should work